### ISSUE TITLE
Procedurally generated plain desktop wallpapers

### ISSUE THREAD

**Issue by velissarious** [helloSystem/hello#185]
Have an option to generate procedurally generated textures like this:
https://en.wikipedia.org/wiki/Procedural_texture

Repeating patterns are a way to have simple wallpapers that are not distracting. They used to be common.
Some pictures of macos 9 desktops I found on internet:
https://blog.alexseifert.com/wp-content/uploads/2014/08/My_Mac_Desktop_Low_Res_and_Cra.jpg
https://upload.wikimedia.org/wikipedia/en/c/cd/Mac_OS_9.0.4_emulated_inside_of_the_SheepShaver_emulator.png

**I would prefer to generate these wallpapers to make them unique and to prevent perfectly repeating patterns.**
Also interesting:
https://en.wikipedia.org/wiki/Aperiodic_tiling
https://en.wikipedia.org/wiki/Tessellation

Although very colorful and spectacular images could be generated I propose more subtle patterns, colors etc in order to be able to clearly see the desktop icons.

Something which I would like the algorithm to be capable to produce:
https://moonavoor.ee/image/type:galleryFullImage/id:79296/filename:XLWS0421_Black_Marble_web.jpg

Another use for such an algorithm would be to generate unique textures for the title bars of windows based on the text of the title (in order to make the window appear the same each time it is opened but different window title bars different). For example the texture of a slightly crumpled paper for text areas, brushed aluminum etc.

**Comment by peteroupc** [comment]
Indeed, classic tiled wallpapers are hard to find nowadays given that desktop backgrounds today tend to cover the full computer screen, to employ thousands of colors, and to have a high-definition resolution (1920×1080 or larger). Essentially, classic tiled wallpapers:

- Are tileable.
- Employ a limited color palette and a limited pixel size.
- Are abstract and do not contain text.
- Are not produced by artificial intelligence tools or with their help.

For details and other guidelines, see my repository on [classic wallpapers](https://github.com/peteroupc/classic-wallpaper).  The repository also houses Python code to generate a tiled wallpaper pattern at random with a custom color palette (such as the 16-color palette for VGA displays, or the so-called "web safe" palette).  The pixel art style is ideal for classic tiled wallpapers, but is not the only possible style.
